state/syncer/userAccountsSyncer.go

```diff
@@ -104,7 +102,7 @@ func NewUserAccountsSyncer(args ArgsNewUserAccountsSyncer) (*userAccountsSyncer,
 }
 
 // SyncAccounts will launch the syncing method to gather all the data needed for userAccounts - it is a blocking method
-func (u *userAccountsSyncer) SyncAccounts(rootHash []byte) error {
+func (u *userAccountsSyncer) SyncAccounts(rootHash []byte, storageMarker common.StorageMarker) error {
 	u.mutex.Lock()
```

Comment:
Please add here a test for nil storageMarker argument. The code can panic on L135. You need to add CheckIfNil on the StorageMarker interface. Valid for validatorAccountsSyncer.go L82. 
+ unit tests

Reply:
Done.